lwc/src/main/scala/quasar/physical/s3/S3DataSource.scala

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2014–2018 SlamData Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package quasar.physical.s3
+
+import eu.timepit.refined.auto._
```

Comment:
We try to structure imports as follows
```
<quasar>

<scala>

<java>

<thirdparty>
```
ordering alphabetically within each section.